language_formatters_pre_commit_hooks/pretty_format_java.py

```diff
@@ -72,7 +74,10 @@ def pretty_format_java(argv: typing.Optional[typing.List[str]] = None) -> int:
     parser.add_argument("filenames", nargs="*", help="Filenames to fix")
     args = parser.parse_args(argv)
 
-    google_java_formatter_jar = __download_google_java_formatter_jar(
+    if Version(args.google_java_formatter_version) <= Version("1.9"):
```

Comment:
Can you add a comment and link to this ref: https://github.com/google/google-java-format/releases/tag/v1.10.0

At first is not clear why you're comparing with `"1.9".